code/__HELPERS/unsorted.dm

```diff
@@ -1566,6 +1566,52 @@ var/mob/dview/tview/tview_mob = new()
 			colour += temp_col
 	return colour
 
+/proc/get_random_potion()	//Pulls up a random potion, excluding minor-types
```

Comment:
Sorry that I didn't mention this before but: The problem I have with this helper is that `get_random_potion()` uses a fixed list to randomize potions, meaning that if somebody in the future decides to add new potions then those won't be immediately added to this list, and now you have a system where adding a potion also requires you to remember to add it to this list. And I don't like that.

A counter-argument would be "but collabs have to review it!", but in reality we can't remember about every thing in the codebase and honestly this can be avoided whatsoever. This also means that overpowered potions would be added to this list too but if that becomes a common thing then we can add a /obj/item/potion/healing/admin child and exclude them from this proc.

How to do it?

```suggestion
/proc/get_random_potion()	//Pulls up a random potion, excluding minor-types
    return pick(subtypesof(/obj/item/potion))
```

If there are potions that shouldn't be here because they are admin-only or some bullshit like that then move those to an /admin child and do

```suggestion
/proc/get_random_potion()	//Pulls up a random potion, excluding minor-types
    return pick(subtypesof(/obj/item/potion) - typesof(/obj/item/potion/admin))
```

That's it, I'll merge your PR after this